src/Tooltip/TooltipDictionary.yaml

```
@@ -6439,6 +6439,323 @@ Bitrise_stepName_StepConfig:
+    width: "400"
+chaos_tune_fault_namespace_selector:
+    content: Labels of the destination namespaces.
+    width: "400"
+chaos_infra_access_type:
+    content: Choose between cluster-wide access and namespace-wide access when setting up a chaos infrastructure.
+    width: "400"
+chaos_infra_namespace:
+    content: Namespace where you wish to install the chaos infrastructure.
+    width: "400"
+chaos_infra_service_acc:
+    content: Service account that you wish to associate with the installed chaos infrastructure.
+    width: "400"
+chaos_infra_basic_config:
+    content: Select the scope of the deployment, the namespace and the service account name.
+    width: "400"
+chaos_infra_adv_config:
+    content: As an advanced user, choose between nodeSelectors (for node affinity) and tolerations (for anti-affinity) that are tunables to adjust the topology.
+    width: "400"
+chaos_infra_selected_connector:
+    content: Connector that you will select to deploy chaos infrastructure and apply the manifest.
+    width: "400"
+chaos_probe_name:
+    content: Name of the probe used to perform chaos experiment health check.
+    width: "400"
+chaos_probe_type:
+    content: Select from four probes- Command, HTTP, K8s, and Prometheus.
+    width: "400"
+chaos_probe_mode:
+    content: Select from five modes- SOT, EOT, Edge, Continuous, and OnChaos.
+    width: "400"
+chaos_probe_timeout:
+    content: Time limit that you set for the probe to execute the specified check and return the result.
+    width: "400"
+chaos_probe_retry:
+    content: Number of times a check is re-run when it fails in the first attempt, before declaring the probe status as failed.
+    width: "400"
+chaos_probe_interval:
+    content: Duration between subsequent retries of a check.
+    width: "400"
+chaos_probe_polling_interval:
+    content: Time interval during which the continuous probe is in sleep state after every iteration.
+    width: "400"
+chaos_probe_initial_delay:
+    content:  Initial waiting time interval for the probe (in seconds).
+    width: "400"
+chaos_probe_stop_failure:
+    content: Choose between true or false to stop or continue the experiment execution after probe failure.
+    width: "400"
+chaos_probe_http_url:
+    content: URL specified to validate the HTTP probe.
+    width: "400"
+chaos_probe_http_skip_cc:
+    content: If the URL specified is secure (that is, HTTPS), check the skip certificate checks checkbox, else uncheck it.
+    width: "400"
+chaos_probe_http_method:
+    content: Choose between HTTP GET and POST methods, which is mapped against a received status code.
+    width: "400"
+chaos_probe_http_criteria:
+    content: Choose between the conditions ==, !=, or oneOf based on the URL and the response code received.
+    width: "400"
+chaos_probe_response_code:
+    content: The value that is mapped against the received status code to validate the probe. If the value is 200, it indicates successful validation of the probe.
```

```suggestion
    content: The value that is mapped against the received status code to validate the probe.
```

200 response code may not be the expected validation criteria at times. Better not to specify this.